src/main/java/juja/sqlcmd/DatabaseManager.java

```diff
@@ -49,6 +49,31 @@ public boolean connect(String database, String user, String password) {
         }
     }
 
+    public boolean insert(String tableName, DataSet dataset) {
+        String insertQueryValues = dataSetCsv(dataset);
+        String sqlQuery = String.format("INSERT INTO %s VALUES(%s)", tableName, insertQueryValues);
+        try(Statement statement = connection.createStatement()){
+            statement.executeUpdate(sqlQuery);
+            return true;
+        } catch (SQLException e) {
+            return false;
+        }
+    }
+
+    private String dataSetCsv(DataSet dataset) {
+        String[] values = dataset.values();
+        StringBuilder csvBuilder = new StringBuilder();
+        for (String value : values) {
+            csvBuilder.append("'")
+                    .append(value)
+                    .append("'")
+                    .append(", ");
+        }
+        int lastCommaIndex = csvBuilder.lastIndexOf(", ");
+        csvBuilder.replace(lastCommaIndex, csvBuilder.length(), "");
+        return csvBuilder.toString();
```

Comment:
You can simplify.
Like this: `return csvBuilder.substring(0,csvBuilder.lastIndexOf(", ")).toString();`

Reply:
@AVM82 Sure, thanks! I will use that. Also in this case ".toString()" is redundant because "substring" returns String by itself